Sources/OFREP/OpenFeatureEvaluationContext+OFREP.swift

```diff
@@ -0,0 +1,53 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OpenFeature open source project
+//
+// Copyright (c) 2025 the Swift OpenFeature project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import OpenAPIRuntime
+import OpenFeature
+
+extension Components.Schemas.Context {
+    package init(_ context: OpenFeatureEvaluationContext) throws {
+        let additionalProperties = try OpenAPIObjectContainer(context.fields)
+        self.init(targetingKey: context.targetingKey, additionalProperties: additionalProperties)
+    }
+}
+
+extension OpenAPIObjectContainer {
+    fileprivate init(_ fields: [String: OpenFeatureFieldValue]) throws {
+        var values = [String: any Sendable]()
+
+        for (key, value) in fields {
+            switch value {
+            case .bool(let value):
+                values[key] = value
+            case .string(let value):
+                values[key] = value
+            case .int(let value):
+                values[key] = value
+            case .double(let value):
+                values[key] = value
+            case .date(let value):
+                values[key] = value.timeIntervalSince1970
+            case .object(let value):
+                let data = try Self.jsonEncoder.encode(value)
+                let container = try Self.jsonDecoder.decode(OpenAPIObjectContainer.self, from: data)
+                values[key] = container.value
```

Comment:
I'm not super happy about this approach to convert an `any Codable & Sendable` into a value to put inside an `OpenAPIObjectContainer`. @czechboy0 Do you have an idea how this could be improved?

Reply:
Thanks for the fast reply 🏎️
`unvalidatedValue` expects a `[String: any Sendable]` dictionary, but `value` in this case is `any Codable & Sendable`.

Reply:
I also tried `OpenAPIValueContainer` but that fails in the `tryCast` because `value` is neither an `array` nor a `dictionary` nor a supported primitive value: https://github.com/apple/swift-openapi-runtime/blob/23146bc8710ac5e57abb693113f02dc274cf39b6/Sources/OpenAPIRuntime/Base/OpenAPIValue.swift#L81

Reply:
Wait, so what is the underlying type? Unless it's a native Swift container or primitive type then yeah, you'll need to run it through Codable.

I wonder why it's Codable though? Where does that requirement come from?

Reply:
It's part of the OpenFeature spec to allow for user-defined objects to be supported in the evaluation API. In the OFREP provider that maps to JSON-coded values but a different provider may encode/decode them differently. That's why I opted for `Codable & Sendable` as the requirement.

Reply:
Perhaps `OpenAPIRuntime` could support "lazy encodable" values in `OpenAPIValueContainer` which would be encoded alongside the request. That way I could save the roundtrip.

Reply:
If that's something you'd like to see, please open an issue on Swift OpenAPI Generator and we can discuss there.